db/db_impl_write.cc

```diff
@@ -184,7 +188,13 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
     // With concurrent writes we do preprocess only in the write thread that
     // also does write to memtable to avoid sync issue on shared data structure
     // with the other thread
+
+    // PreprocessWrite does its own perf timing.
+    PERF_TIMER_STOP(write_pre_and_post_process_time);
```

Comment:
can we mention the ones that were added or changed in `HISTORY.md`?

Reply:
hmm I still think we should mention the timers whose meanings were changed by this PR, like `write_pre_and_post_process_time`. It'll be non-obvious to users why times increased or decreased when upgrading their rocksdb version, if people pay attention to these timers.